csrc/quantization/machete/machete_prepacked_layout.cuh

```diff
@@ -0,0 +1,199 @@
+#pragma once
+
+#include <ATen/cuda/CUDAContext.h>
+#include <c10/cuda/CUDAGuard.h>
+#include <torch/all.h>
+
+// clang-format off
+// The cutlass include order 
```

Comment:
nit: fixup comment "The cutlass include order matters"

Reply:
done